BouncyCastle-JCA/src/KeyFactory.crysl

```diff
@@ -12,14 +14,17 @@ EVENTS
     gPriv: generatePrivate(keySpec);
     gPubl: generatePublic(keySpec);
     
-    Gens := gPriv | gPubl;
-
 ORDER
-    Gets, Gens
+    Gets, (gPriv* | gPubl*)*
```

Comment:
Why did you change this?

Reply:
Here I used the fix that was introduced in the JCA's [KeyFactory](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/JavaCryptographicArchitecture/src/KeyFactory.crysl#L17) rule and I am not sure why this was changed there. My guess would be that this was changed to ensure the last predicates in the [KeyFactory](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/JavaCryptographicArchitecture/src/KeyFactory.crysl#L27) rule.